packages/amplify-cli/src/extensions/amplify-helpers/current-cloud-backend-utils.ts

```diff
@@ -1,9 +1,8 @@
 import { $TSAny, $TSContext, AmplifyFault, spinner, stateManager } from '@aws-amplify/amplify-cli-core';
 import sequential from 'promise-sequential';
-import { notifyFieldAuthSecurityChange, notifyListQuerySecurityChange, notifySecurityEnhancement } from './auth-notifications';
 import { getProviderPlugins } from './get-provider-plugins';
 import { getProjectConfig } from './get-project-config';
-
+import { checkForcedUpdates as checkApiForcedUpdates } from '@aws-amplify/amplify-category-api';
```

Comment:
thought for future. if we ever get to `ApiCategory implements ICategory` then there should be `ICategory.checkForcedUpdate` with default no op impl. or some cross cutting mechanism.